test/docker-outside-of-docker/scenarios.json

```diff
@@ -76,26 +77,6 @@
         },
         "containerUser": "vscode"
     },
-    "docker_dash_compose_v1": {
```

Comment:
> You would be able to revert these changes after you add back compose switch installation! 

Can we add a test scenario which validates specific version/semver changes? Thanks!